src/lib/utils/events.ts

```diff
@@ -1,6 +1,14 @@
 import { deriveFormattedTimestamp } from './date'
 
 export function formatEventCTA(input: string): string {
+  if (!input) {
```

Comment:
Extra null check here; but this function should not be called if the input is falsey.